### ISSUE TITLE
Some question about  scheduler plugin config

### ISSUE THREAD

**Issue by hyc-yuchen** [hkust-adsl/kubernetes-scheduler-simulator#11]
Hello, I found that there is a "-gpusel FGD" parameter in the FGD scheduler parameter when I looked at the training script, but I looked at the corresponding code in generate_config_and_run.py and found that it is only "best, worst, random", may I ask how this is added?
![image](https://github.com/hkust-adsl/kubernetes-scheduler-simulator/assets/61409162/814b716c-6f13-4411-b3c9-54b351cc745e)
![image](https://github.com/hkust-adsl/kubernetes-scheduler-simulator/assets/61409162/27b7c6a6-6e21-4c3f-b4ac-bab1a230c834)

**Comment by qzweng** [comment]
Hi, yuchen @hyc-yuchen . Thanks for your question.

`gpusel` stands for "GPU Selection Method", which determines how the plugin extends the *node-selection* scheduling into a finer-grained *GPU-selection* scheduling.  Yet, our proposed *FGD* (fragmentation gradient descent) is natively a *GPU-selection* scheduling policy, thus its GPU selection method (`-gpusel FGD`) not applicable to other baseline scheduling policies.

In short, the python `generate_config_and_run.py` would use the plugin name (FGD) as gpu_sel_method for FGD (see [L#268](https://github.com/hkust-adsl/kubernetes-scheduler-simulator/blob/main/scripts/generate_config_and_run.py#L268)). For plugins needing specifying gpu_sel_method, see its implementation in `open_gpu_share.go` starting from [L#268](https://github.com/hkust-adsl/kubernetes-scheduler-simulator/blob/main/pkg/simulator/plugin/open_gpu_share.go#L268).
